Colours_And_Fonts_style_v1.css

```diff
@@ -198,17 +288,24 @@ ul li:hover ul {
 
 .content{
 	box-sizing: border-box;
-	background-color: #219ebc;
+	background-color: #FFFFFF;

```

Comment:
I changed the content colour from a drak blue to white so the end users can see the text better as the darks were to dark for the end users eyes.